src/app/item-feed/Highlights.tsx

```diff
@@ -60,11 +64,40 @@ export default function Highlights({ item }: { item: DimItem }) {
         {stat.value}
       </div>
     );
+    // exotic class armor intrinsics
+    const extraIntrinsicSockets =
+      item.isExotic && item.bucket.hash === BucketHashes.ClassArmor && item.sockets
```

Comment:
Maybe worth pulling this logic into a `canHaveExtraIntrinsics` function.

Reply:
Making it getExtraIntrinsicPerkSockets in socket-utils.